crates/red_knot_python_semantic/src/types/unpacker.rs

```diff
@@ -35,16 +35,25 @@ impl<'db> Unpacker<'db> {
         self.context.db()
     }
 
-    /// Unpack the value to the target expression.
     pub(crate) fn unpack(&mut self, target: &ast::Expr, value: UnpackValue<'db>) {
         debug_assert!(
             matches!(target, ast::Expr::List(_) | ast::Expr::Tuple(_)),
-            "Unpacking target must be a list or tuple expression"
+            "Unpacking target must be a list or tuple"
```

Comment:
This is a minor nit, but I don't think we should remove the word "expression" here; the LHS isn't really a list or tuple (no such object is ever constructed), it's just an expression in that syntactic form
```suggestion
            "Unpacking target must be a list or tuple expression"
```